Tasks/KubernetesV1/src/kubernetescommand.ts

```diff
@@ -26,6 +26,7 @@ function getCommandOutputFormat(kubecommand: string): string[] {
     var args: string[] = [];
     var outputFormat = tl.getInput("outputFormat", false);
     switch (outputFormat) {
+        case null:
```

Comment:
Is this case even being hit?

Reply:
Yes. In case of empty string for outputFormat in yaml.

Reply:
In case of empty string, shouldn't  `''` case be hit?

Reply:
No, that's the reason it is failing.